litestar/contrib/pydantic/utils.py

```diff
@@ -116,9 +114,6 @@ def pydantic_unwrap_and_get_origin(annotation: Any) -> Any | None:
         return get_origin_or_inner_type(annotation)
 
     origin = annotation.__pydantic_generic_metadata__["origin"]
-    if origin in wrapper_type_set:
-        inner, _, _ = unwrap_annotation(annotation)
-        origin = get_origin_or_inner_type(inner)
```

Comment:
@guacs can you review this?

I couldn't get coverage on these lines b/c the wrapper types (e.g., `Annotated`) don't have a `__pydantic_generic_metadata__` attribute. I.e., sending in `Annotated[SomePydanticModel, ...]` just fails with attribute error.

If removing the lines isn't the right thing to do, the function will need a refactor.

Reply:
I think this should be fine since we're moving all the parsing of the `Annotated` logic to `FieldDefinition`. I'm assuming by the time this function ever gets called, the `Annotated` would have been stripped.

Reply:
sweet!